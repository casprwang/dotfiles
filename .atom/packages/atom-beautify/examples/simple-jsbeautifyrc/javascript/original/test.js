function hell() {
console.log('world');
}
