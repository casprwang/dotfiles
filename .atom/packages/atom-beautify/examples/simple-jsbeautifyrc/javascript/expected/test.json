{
  "indent_size": 2,
  "indent_char": " ", // test
  "indent_level": 0,
  "indent_with_tabs": false,
  /* test */
  "preserve_newlines": true,
  "max_preserve_newlines": 2,
  /* test *
  "jslint_happy": true
}